src/components/Header/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {useNavigate} from 'react-router-dom';
+import { 
+    HeaderContainer,
+    NavigationHeader,
+    HeaderBackButton,
+    HeaderTitle,
+} from './Header.styled';
+
+interface HeaderProps {
```

Comment:
Renamed from `Props` to `HeaderProps` , to specify and easy the reading of the component